README.rst

```diff
@@ -294,7 +294,7 @@ but this can easily be changed to your favorite framework, such as TensorFlow, o
         pred = model(x)
 
         # compute loss and gradients
-        loss, grads = ivy.execute_with_gradients(lambda params: loss_fn(*params), (model.v, x, y), xs_grad_idxs=[[0]])
+        loss, grads = ivy.execute_with_gradients(lambda params: loss_fn(*params), (model.v, x, y))
 
         # update parameters
         model.v = optimizer.step(model.v, grads)
```

Comment:
I'm getting this
```bash
    model.v = optimizer.step(model.v, grads)
  File "/workspace/graph-compiler/ivy_repo/ivy/stateful/optimizers.py", line 157, in step
    return self._step_fn(v, grads, ignore_missing)
  File "/workspace/graph-compiler/ivy_repo/ivy/stateful/optimizers.py", line 112, in _step_fn
    return self._step(v, grads)
  File "/workspace/graph-compiler/ivy_repo/ivy/stateful/optimizers.py", line 386, in _step
    self._vw = grads**2

TypeError: unsupported operand type(s) for ** or pow(): 'list' and 'int'
```
on latest master, would be good to look into this before merging :+1:

Reply:
the rest looks good to me :+1:

Reply:
No worries! I'll get the PR merged in that case, thanks @guillesanbri 😄